graphql-query-language/notes/05-filter-a-qraphql-query-result-using-qrguments.md

```diff
@@ -8,7 +8,13 @@ In this lesson, we will filter a list of pets by status.
 
 We've gotten total pets. Total pets tells us that there are 25 pets that are part of our library, but I might want to filter this list to see just the pets that are available or just the pets that are checked out.
 
-To do this, I'm going to add a GraphQL argument. There's an argument on the `totalPets` query called `status`. This will take in either available or checked out. If we add Available, we'll see that there are 20 available pets.
+To do this, I'm going to add a GraphQL argument. There's an argument on the `totalPets` query called `status`. This will take in either available or checked out. If we add `AVAILABLE`, we'll see that there are 20 available pets.
+
+<TimeStamp start="0:24" end="0:30">
+
```

Comment:
I like these notes because they allow the user to copy-paste what's being shown on the screen.

Reply:
But at the same time, I'm wondering how valuable they'd actually be since the code is already on the screen. If the user is able to save notes they'd also be missing context

Reply:
I think we should see how it actually feels while watching the lesson